src/components/SignInButtons/AppleSignIn/index.website.js

```diff
@@ -105,17 +115,16 @@ AppleSignInDiv.propTypes = requiredPropTypes;
 // The Sign in with Apple script may fail to render button if there are multiple
 // of these divs present in the app, as it matches based on div id. So we'll
 // only mount the div when it should be visible.
-function SingletonAppleSignInButton({isDesktopFlow}) {
-    const isFocused = useIsFocused();
+function SingletonAppleSignInButton({isFocused, isDesktopFlow}) {
     if (!isFocused) {
         return null;
     }
     return <AppleSignInDiv isDesktopFlow={isDesktopFlow} />;
 }
 
-SingletonAppleSignInButton.propTypes = requiredPropTypes;
+SingletonAppleSignInButton.propTypes = singletonPropTypes;
 
-function AppleSignIn({isDesktopFlow}) {
+function AppleSignIn({isDesktopFlow, isFocused}) {
```

Comment:
Nit, but I don't think we need to pass `isFocused` in here. You can just make a wrapped version of `Singleton...`

```
const SingletonAppleSignInButtonWithFocus = withNavigationFocus(SignletonAppleSignInButton)

...

return <SingletonAppleSignInButtonWithFocus isDesktopFlow={isDesktopFlow} />
```

Name is horrendous, but that's the idea :)